src/dir.proj

```diff
@@ -67,6 +71,14 @@
     <RemoveDir  Directories="Agent.Listener/obj" />
     <RemoveDir  Directories="Agent.Worker/bin" />
     <RemoveDir  Directories="Agent.Worker/obj" />
+    <RemoveDir  Directories="Agent.PluginHost/bin" />
+    <RemoveDir  Directories="Agent.PluginHost/obj" />
+    <RemoveDir  Directories="Agent.PluginCore/bin" />
+    <RemoveDir  Directories="Agent.PluginCore/obj" />
+    <RemoveDir  Directories="Agent.RepositoryPlugin/bin" />
+    <RemoveDir  Directories="Agent.RepositoryPlugin/obj" />
+    <RemoveDir  Directories="Agent.DropPlugin/bin" />
```

Comment:
would it be a worthy reduction in overhead to just have two DLLs? PluginHost and Plugins? and let authors namespace their code.

We will quickly hit an issue where folks want shared DLLs for shared logic. That problem goes away if all plugsin are in one DLL. And note, everything is in one DLL today in the worker project.

Reply:
agent.pluginhost.exe
agent.pluginsdk.dll
agent.plugins.dll  <- microsoft.vs.service.agent.plugins.repo/drop/etc...